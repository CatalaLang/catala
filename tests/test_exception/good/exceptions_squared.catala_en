## Test

```catala
declaration scope A:
  context x content integer

scope A:
  label base_x
  definition x equals 0

  label exception_x
  exception base_x
  definition x equals 1

  exception exception_x
  definition x equals 2
```
