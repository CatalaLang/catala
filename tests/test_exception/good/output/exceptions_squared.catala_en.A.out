[RESULT] Computation successful! Results:
[RESULT] x = 2
